collect_app/src/main/res/layout/sort_item_layout.xml

```diff
@@ -14,12 +14,13 @@ License for the specific language governing permissions and limitations under
 the License.
 
 -->
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Why `FrameLayout`?

Reply:
I found that `android:foreground` attribute does not work in API 23. I read about it [here](https://stackoverflow.com/questions/52329249/why-doesnt-androidforeground-attribute-work). Including it in FrameLayout works.

Reply:
Even if it's true it's not a related change so you can file a separate issue for that. You can't just change `LineraLayout` to `FrameLayout` because those component's work in a different way.